ui/nuxeo-path-suggestion/nuxeo-path-suggestion.js

```diff
@@ -174,24 +173,6 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
       };
     }
 
-    ready() {
-      super.ready();
-      const validateFn = this._validate.bind(this);
-      const validator = {
```

Comment:
Passing the validator like this doesn't work. Different elements will have the same validator and reference the same element (the `this`), so this doesn't work. We can go through an interactive review if it makes sense.